### DISCUSSION TITLE
Error running the R code in Section 2.5 Seasonal Subseries Plot

### DISCUSSION THREAD

**Discussion by ogamonal** [OTexts/Discussion#4] [Error report]
In section 2.5 Seasonal Subseries Plot, if I  generate the HTML (Knit to HTML) of "Example: Australian holiday tourism", the result is correct, but if I try to run the code in RStudio (R Markdown), I receive a message that says:

Error in `autoplot()`:
! Objects of class <tbl_df> are not supported by autopilot.

Please, can you help me?

Thank you.

**Comment by robjhyndman** [comment]
You will have modified the object so that it is no longer a tsibble. The output says that it is a tibble object, which does not have an autoplot method.
